src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/controller/PartnershipController.java

```diff
@@ -98,11 +100,32 @@ public ResponseEntity<Object> updatePartnership(
         }
     }
 
+    @GetMapping("/{" +URL_PARAM_SUBMISSION_ID + "}")
+    public ResponseEntity<Object> getPartnership(
```

Comment:
Should this be called `getLimitedPartnership`?

Reply:
It's using the same naming convention as the POST (createPartnership) and PATCH (updatePartnership). Based on our conversations with Mike this will only pull back partnership details.